packages/manager/modules/pci/src/projects/project/users/roles/roles.routing.js

```diff
@@ -0,0 +1,24 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('pci.projects.project.users.roles', {
+    url: '/roles?userId',
+    views: {
+      modal: {
+        component: 'pciProjectsProjectUsersRoles',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      breadcrumb: /* @ngInject */ () => null,
+      goBack: /* @ngInject */ goToUsers => goToUsers,
```

Comment:
Where is it defined ? 🤔

Reply:
In users.routing, it is an injected resolve value